packages/deployer/subtasks/validate-router.js

```diff
@@ -28,7 +30,10 @@ subtask(
 async function _runSourceValidations() {
   const errorsFound = [];
 
-  const validator = new RouterSourceValidator();
+  const validator = new RouterSourceValidator({
+    getModulesSelectors,
+    getRouterSource,
+  });
```

Comment:
I had to move this here so we were able to unit test the validator helper without having to mock an entire project with build and everything.